docs/getting-started.md

```diff
@@ -106,30 +79,21 @@ Congratulations you are still with us and you have build KubeVirt.
 
 Now it's time to get hands on and give it a try.
 
-### Cockpit
-
-Cockpit is exposed on <http://192.168.200.2:9090>
-The default login is `root:vagrant`
-
-It can be used to view the cluster and verify the running state of
-components within the cluster.
-More information can be found on that [project's site](http://cockpit-project.org/guide/latest/feature-kubernetes.html).
-
 ### Create a first Virtual Machine
 
-Finally start a VM called `testvm`:
+Finally start a VM called `vm-ephemeral`:
 
 ```bash
     # This can be done from your GIT repo, no need to log into a vagrant VM
 
     # Create a VM
-    ./cluster/kubectl.sh create -f cluster/vm.yaml
+    ./cluster/kubectl.sh create -f cluster/vms-examples/vm-ephemeral.yaml
```

Comment:
you'll want to 's/vms-examples/examples'

Reply:
thanks missed it